app/templates/components/vnc-viewer.emblem

```diff
@@ -12,7 +12,7 @@ if isPoppedOut
         if isNotTablet
           div class="speaker"
       div class=screenRequired:noscreen:screen
-        canvas.canvas
+        div.canvas
```

Comment:
name the div as `screen` or something rather than canvas. It is very confusing

Reply:
div as screen? screen is not an element. we can rather name the class as screen, but already have screen which contains the canvas. we can name the class as canvas-container or something like that